CHANGES.md

```diff
@@ -28,6 +28,18 @@ OpenSSL 3.3
 
 ### Changes between 3.2 and 3.3 [xx XXX xxxx]
 
+ * Unknown entries in TLS SignatureAlgorithms, ClientSignatureAlgorithms
+   config options and the respective calls to SSL[_CTX]_set1_sigalgs() and
+   SSL[_CTX]_set1_client_sigalgs() that are marked with `?` character are
```

Comment:
s/are marked with/start with/ here and below?

Reply:
fixed